src/transformers/training_args.py

```diff
@@ -138,6 +138,10 @@ class TrainingArguments:
             Number of predictions steps to accumulate the output tensors for, before moving the results to the CPU. If
             left unset, the whole predictions are accumulated on GPU/TPU before being moved to the CPU (faster but
             requires more memory).
+        num_eval_delay_epochs (`float`, *optional*):
+            Number of epochs to wait for before the first evaluation can be performed.
+        num_eval_delay_steps (`int`, *optional*):
+            Number of steps to wait for before the first evaluation can be performed. Overrides num_eval_delay_epochs.
```

Comment:
I liked you initial suggestion of having one argument `eval_delay` (which would be steps or epochs depending on the strategy) better. There is already 80 training arguments ;-)

Reply:
I was thinking people might want a mix but I can't actually think of an example when!